services/scabbard/libscabbard/src/service/shared.rs

```diff
@@ -114,7 +183,9 @@ impl ScabbardShared {
             );
         }
 
-        batch
+        self.update_accepting_batches()?;
```

Comment:
Should this also be added to the `add_batch_to_queue` function?

Reply:
I reworked the solution a little bit to move the updating_accepting_batches into add_batch_to_queue/pop_batch_from_queue only